src/tokenOperations/tokenOperationsFactory.test.net.spec.ts

```diff
@@ -0,0 +1,500 @@
+        });
+
+        const tx1OnNetwork = await processTransaction(frank, tx1, "tx1");
+        const tx1Outcome = parser.parseIssueFungible(tx1OnNetwork);
+        const tokenIdentifier = tx1Outcome.tokenIdentifier;
+        assert.isTrue(tokenIdentifier.includes("FRANK"));
+
+        // Set roles (give Grace the ability to mint and burn)
+        const tx2 = factory.setSpecialRoleOnFungible({
+            manager: frank.address,
+            user: grace.address,
+            tokenIdentifier: tx1Outcome.tokenIdentifier,
+            addRoleLocalMint: true,
+            addRoleLocalBurn: true,
+            transactionNonce: frank.account.nonce
+        });
+
+        const tx2OnNetwork = await processTransaction(frank, tx2, "tx2");
+        const tx2Outcome = parser.parseSetSpecialRole(tx2OnNetwork);
+        assert.include(tx2Outcome.roles, "ESDTRoleLocalMint");
+        assert.include(tx2Outcome.roles, "ESDTRoleLocalBurn");
+
+        // Mint (Grace mints for herself)
+        const tx3 = factory.localMint({
+            manager: grace.address,
+            user: grace.address,
+            tokenIdentifier: tokenIdentifier,
+            supplyToMint: 200,
+            transactionNonce: grace.account.nonce
+        });
+
+        const tx3OnNetwork = await processTransaction(grace, tx3, "tx3");
+        const tx3Outcome = parser.parseLocalMint(tx3OnNetwork);
+        assert.equal(tx3Outcome.mintedSupply, "200");
+
+        // Burn (Grace burns 50 of her tokens)
+        const tx4 = factory.localBurn({
+            manager: grace.address,
+            user: grace.address,
+            tokenIdentifier: tokenIdentifier,
+            supplyToBurn: 50,
+            transactionNonce: grace.account.nonce
+        });
+
+        const tx4OnNetwork = await processTransaction(grace, tx4, "tx4");
+        const tx4Outcome = parser.parseLocalBurn(tx4OnNetwork);
+        assert.equal(tx4Outcome.burntSupply, "50");
+    });
+
+    it("should issue fungible, pause, unpause", async function () {
+        this.timeout(240000);
+        await frank.sync(provider);
+
+        // Issue
+        const tx1 = factory.issueFungible({
+            issuer: frank.address,
+            tokenName: "FRANK",
+            tokenTicker: "FRANK",
+            initialSupply: 100,
+            numDecimals: 0,
+            canFreeze: true,
+            canWipe: true,
+            canPause: true,
+            canMint: true,
+            canBurn: true,
+            canChangeOwner: true,
+            canUpgrade: true,
+            canAddSpecialRoles: true,
+            transactionNonce: frank.account.nonce
+        });
+
+        const tx1OnNetwork = await processTransaction(frank, tx1, "tx1");
+        const tx1Outcome = parser.parseIssueFungible(tx1OnNetwork);
+        const tokenIdentifier = tx1Outcome.tokenIdentifier;
+        assert.isTrue(tokenIdentifier.includes("FRANK"));
+
+        // Pause
+        const tx2 = factory.pause({
+            pause: true,
```

Comment:
instead of a function `pause` with `pause` and `unpause` arguments, the function could have been called `pauseOperation` and it can have an argument `type` of an enum type that has `PAUSE / UNPAUSE`. 
This is an optional refactoring, but if applicable, please apply it for freeze/wipe as well

Reply:
Fixed, a bit differently - using separate functions now.